inst/templates/tidy-eval.R

```diff
@@ -1,15 +1,44 @@
 #' Tidy eval helpers
 #'
-#' These functions provide tidy eval-compatible ways to capture
-#' symbols (`sym()`, `syms()`, `ensym()`), expressions (`expr()`,
-#' `exprs()`, `enexpr()`), and quosures (`quo()`, `quos()`, `enquo()`).
-#' To learn more about tidy eval and how to use these tools, read
-#' <http://rlang.tidyverse.org/articles/tidy-evaluation.html>
+#' @description
```

Comment:
I tried to do what @lionel- said here:

https://github.com/r-lib/usethis/pull/377#issuecomment-393437046